peer-ids/peer-ids.md

```diff
@@ -0,0 +1,95 @@
+# Spec:  Peer Ids and Keys
+
+## Keys
+
+
+Our key pairs are transmitted on the wire using a simple protobuf defined in [libp2p/go-libp2p-crypto/pb/crypto.proto#L5](https://github.com/libp2p/go-libp2p-crypto/blob/master/pb/crypto.proto#L5):
+
+```
```

Comment:
perhaps add a `protobuf` after the backticks so that it is syntax highlighted.

Reply:
Good point - thanks!  Fixed